src/librustc/lint/mod.rs

```diff
@@ -97,12 +97,13 @@ macro_rules! declare_lint {
 
 /// Declare a static `LintArray` and return it as an expression.
 #[macro_export]
-macro_rules! lint_array { ($( $lint:expr ),*) => (
-    {
-        static ARRAY: LintArray = &[ $( &$lint ),* ];
-        ARRAY
-    }
-) }
+macro_rules! lint_array {
+    ($( $lint:expr ),*,) => { lint_array!( $( &$lint ),* ) };
```

Comment:
Why did you add a `&` here?

Reply:
Oh, good catch... Copy/paste :face_with_head_bandage:

Reply:
Fixed. Thanks @durka !